Scripts/Python/xLinkingBookGUIPopup.py

```diff
@@ -490,7 +490,7 @@ def IShowBookNoTreasure(self):
                     sdl["CleftVisited"] = (1,)
                     vault.addChronicleEntry("TomahnaLoad",1,"yes")
                     PtDebugPrint("Chronicle entry TomahnaLoad not present, adding entry and setting to yes")
-                respLinkResponder.run(self.key,avatar=PtGetLocalAvatar())
+                respLinkResponder.run(self.key,avatar=PtGetLocalAvatar(),netPropagate=0)
```

Comment:
FWIW, this call didn't have `netPropagate=0` before, but adding it shouldn't hurt probably.

Reply:
Your right, I can remove if its a problem.